cmd/gb-vendor/fetch.go

```diff
@@ -56,7 +56,11 @@ Flags:
 			return err
 		}
 
-		wc, err := repo.Checkout(branch, revision)
+		if m.HasDependencyWithUrl(repo.URL()) {
+			return fmt.Errorf("specified repository is already downloaded")
```

Comment:
For some reason I cannot find these comments on github, which makes it hard
to understand the context.

On Sun, Jun 7, 2015 at 7:57 PM, Dávid Kaya <notifications@github.com> wrote:

> In cmd/gb-vendor/fetch.go
> <https://github.com/constabulary/gb/pull/192#discussion_r31875466>:
>
> > @@ -56,7 +56,11 @@ Flags:
> >  			return err
> >  		}
> >
> > -		wc, err := repo.Checkout(branch, revision)
> > +		if m.HasDependencyWithUrl(repo.URL()) {
> > +			return fmt.Errorf("specified repository is already downloaded")
>
> I am afraid I don't quite understand you :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/constabulary/gb/pull/192/files#r31875466>.
>